Formula/orientdb.rb

```diff
@@ -1,9 +1,9 @@
 class Orientdb < Formula
   desc "Graph database"
   homepage "https://orientdb.com"
-  url "https://orientdb.com/download.php?email=unknown@unknown.com&file=orientdb-community-2.1.16.tar.gz&os=mac"
-  version "2.1.16"
-  sha256 "41ad0db53c418459d0efbf6a7f7e2b39f48467f1ec582efa925ceb38de3f3cc6"
+  url "http://orientdb.com/download.php?email=unknown@unknown.com&file=orientdb-community-2.2.2.tar.gz&os=mac"
```

Comment:
Please don't switch back to insecure HTTP if HTTPS works. It also looks like the URL can be simplified (according to the information in the OrientDB documentation):

```ruby
url "https://orientdb.com/download.php?file=orientdb-community-2.2.2.tar.gz"
```